grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java

```diff
@@ -219,38 +239,58 @@ void sendQueriesToLoader(Collection<Query> queries){
             throw new RuntimeException(e);
         }
 
+
+        Json configuration = Json.object()
+                .set(KEYSPACE_PARAM, keyspace)
+                .set(BATCH_NUMBER, batchNumber)
+                .set(TASK_LOADER_MUTATIONS,
+                        queries.stream().map(Query::toString).collect(toList()));
+
+        Callable<TaskId> callable = () -> taskClient
+                .sendTask("ai.grakn.engine.loader.MutatorTask",
+                        BatchMutatorClient.class.getName(),
+                        Instant.ofEpochMilli(new Date().getTime()), null, configuration, 10000);
+
+        Retryer<TaskId> retryer = RetryerBuilder.<TaskId>newBuilder()
```

Comment:
Previously we would only re-attempt to send the task if the `retry` boolean was true, but now it looks like we are always doing it?